### PR TITLE
BI-3972 create objection information page

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: src/controllers/enter.information.controller.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+
+const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```
Comment (ICoxCH) [inline src/controllers/enter.information.controller.ts:4]: Out of interest does this have to be an async function with a promise?

Comment (markpit) [reply src/controllers/enter.information.controller.ts:4]: I think you only need the function to be async if you are await'ing something inside. in this case you can take off the async + Promise<void>.   You should be able to just use (blah, blah, blah) => {

File: src/controllers/enter.information.controller.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+
+const route = (req: Request, res: Response, next: NextFunction) => {
+  return res.render(Templates.ENTER_INFORMATION, {
+    templateName: Templates.ENTER_INFORMATION,
```
[the remark this answers is not in the document]
Comment (SVarallo) [reply src/controllers/enter.information.controller.ts:6]: I'm not sure reason is the right word to use either, I agree it is a bit vague though. I'll discuss with Oceanne.

[the remark this answers is not in the document]
Comment (SVarallo) [reply src/controllers/enter.information.controller.ts:6]: Oceanne is of the mind that enter-information is explanatory enough,  the word reason implies that you might have more than one. The user testing today might come back with some opinions on the page though so we’ll see about changing it in the future if the need arises.

**Pull request merged**